isminet/clients/base.py

```
@@ -1,168 +1,182 @@
+
+        return data
+
+    def _handle_error_response(self, response: Response) -> None:
+        """Handle error response."""
+        error_msg = response.text
+        try:
+            error_data = response.json()
+            if "meta" in error_data and "msg" in error_data["meta"]:
+                error_msg = error_data["meta"]["msg"]
+        except Exception:
+            pass
```

Ruff (0.8.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)